src/config/selectors.ts

```diff
@@ -22,8 +31,19 @@ export const SUMMON_BUTTON_CY = 'summon_button';
 export const BANNER_CY = 'banner';
 export const KEYWORD_BUTTON_CY = 'keyword_button';
 
+export const CHAT_WINDOW_CY = 'chat_window';
+export const DICTIONNARY_MODE_CY = 'dictionnary_mode';
+export const CHATBOT_MODE_CY = 'chatbot_mode_cy';
+
 export const buildDataCy = (selector: string): string =>
   `[data-cy=${selector}]`;
 
 export const buildTextFieldSelectorCy = (selector: string): string =>
   `${buildDataCy(selector)} input`;
+
+export const keywordDataCy = (keyword: string): string =>
+  `${keyword}-keyword_button`;
```

Comment:
You can factor `-keyword_button` out to a constant and re-use it here and in the partial selector you have on line 49 !